Failed to load native library:sqlite-3.40.0.0-f39c7234-f031-4165-a2b1-0d85bcea48a1-libsqlitejdbc.so. osinfo: Linux/x86_64

Just updated signal-cli via the Debian repository and getting this error:

```
Failed to load native library:sqlite-3.40.0.0-f39c7234-f031-4165-a2b1-0d85bcea48a1-libsqlitejdbc.so. osinfo: Linux/x86_64
java.lang.UnsatisfiedLinkError: Can't load library: /tmp/sqlite-3.40.0.0-f39c7234-f031-4165-a2b1-0d85bcea48a1-libsqlitejdbc.so
	at org.graalvm.nativeimage.builder/com.oracle.svm.core.jdk.NativeLibrarySupport.loadLibraryAbsolute(NativeLibrarySupport.java:92)
	at java.base@17.0.5/java.lang.ClassLoader.loadLibrary(ClassLoader.java:57)
	at java.base@17.0.5/java.lang.Runtime.load0(Runtime.java:755)
	at java.base@17.0.5/java.lang.System.load(System.java:1953)
	at org.sqlite.SQLiteJDBCLoader.loadNativeLibrary(SQLiteJDBCLoader.java:280)
	at org.sqlite.SQLiteJDBCLoader.extractAndLoadLibraryFile(SQLiteJDBCLoader.java:237)
	at org.sqlite.SQLiteJDBCLoader.loadSQLiteNativeLibrary(SQLiteJDBCLoader.java:350)
	at org.sqlite.SQLiteJDBCLoader.initialize(SQLiteJDBCLoader.java:68)
	at org.sqlite.core.NativeDB.load(NativeDB.java:63)
	at org.sqlite.SQLiteConnection.open(SQLiteConnection.java:278)
	at org.sqlite.SQLiteConnection.<init>(SQLiteConnection.java:65)
	at org.sqlite.jdbc3.JDBC3Connection.<init>(JDBC3Connection.java:28)
	at org.sqlite.jdbc4.JDBC4Connection.<init>(JDBC4Connection.java:19)
	at org.sqlite.JDBC.createConnection(JDBC.java:104)
	at org.sqlite.JDBC.connect(JDBC.java:77)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:121)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:359)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:201)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:470)
	at com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:561)
	at com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:100)
	at com.zaxxer.hikari.HikariDataSource.<init>(HikariDataSource.java:81)
	at org.asamk.signal.manager.storage.Database.getHikariDataSource(Database.java:101)
	at org.asamk.signal.manager.storage.Database.initDatabase(Database.java:32)
	at org.asamk.signal.manager.storage.AccountDatabase.init(AccountDatabase.java:32)
	at org.asamk.signal.manager.storage.SignalAccount.lambda$getAccountDatabase$38(SignalAccount.java:1236)
	at org.asamk.signal.manager.storage.SignalAccount.getOrCreate(SignalAccount.java:1649)
	at org.asamk.signal.manager.storage.SignalAccount.getAccountDatabase(SignalAccount.java:1234)
	at org.asamk.signal.manager.storage.SignalAccount.lambda$getRecipientStore$30(SignalAccount.java:1209)
	at org.asamk.signal.manager.storage.SignalAccount.getOrCreate(SignalAccount.java:1649)
	at org.asamk.signal.manager.storage.SignalAccount.getRecipientStore(SignalAccount.java:1206)
	at org.asamk.signal.manager.storage.SignalAccount.load(SignalAccount.java:632)
	at org.asamk.signal.manager.storage.SignalAccount.load(SignalAccount.java:181)
	at org.asamk.signal.manager.SignalAccountFiles.initManager(SignalAccountFiles.java:92)
	at org.asamk.signal.manager.SignalAccountFiles.initManager(SignalAccountFiles.java:79)
	at org.asamk.signal.App.loadManager(App.java:355)
	at org.asamk.signal.App.handleLocalCommand(App.java:288)
	at org.asamk.signal.App.init(App.java:229)
	at org.asamk.signal.Main.main(Main.java:60)
ERROR HikariPool - HikariPool-1 - Exception during pool initialization.
java.sql.SQLException: Error opening connection
	at org.sqlite.SQLiteConnection.open(SQLiteConnection.java:281)
	at org.sqlite.SQLiteConnection.<init>(SQLiteConnection.java:65)
	at org.sqlite.jdbc3.JDBC3Connection.<init>(JDBC3Connection.java:28)
	at org.sqlite.jdbc4.JDBC4Connection.<init>(JDBC4Connection.java:19)
	at org.sqlite.JDBC.createConnection(JDBC.java:104)
	at org.sqlite.JDBC.connect(JDBC.java:77)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:121)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:359)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:201)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:470)
	at com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:561)
	at com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:100)
	at com.zaxxer.hikari.HikariDataSource.<init>(HikariDataSource.java:81)
	at org.asamk.signal.manager.storage.Database.getHikariDataSource(Database.java:101)
	at org.asamk.signal.manager.storage.Database.initDatabase(Database.java:32)
	at org.asamk.signal.manager.storage.AccountDatabase.init(AccountDatabase.java:32)
	at org.asamk.signal.manager.storage.SignalAccount.lambda$getAccountDatabase$38(SignalAccount.java:1236)
	at org.asamk.signal.manager.storage.SignalAccount.getOrCreate(SignalAccount.java:1649)
	at org.asamk.signal.manager.storage.SignalAccount.getAccountDatabase(SignalAccount.java:1234)
	at org.asamk.signal.manager.storage.SignalAccount.lambda$getRecipientStore$30(SignalAccount.java:1209)
	at org.asamk.signal.manager.storage.SignalAccount.getOrCreate(SignalAccount.java:1649)
	at org.asamk.signal.manager.storage.SignalAccount.getRecipientStore(SignalAccount.java:1206)
	at org.asamk.signal.manager.storage.SignalAccount.load(SignalAccount.java:632)
	at org.asamk.signal.manager.storage.SignalAccount.load(SignalAccount.java:181)
	at org.asamk.signal.manager.SignalAccountFiles.initManager(SignalAccountFiles.java:92)
	at org.asamk.signal.manager.SignalAccountFiles.initManager(SignalAccountFiles.java:79)
	at org.asamk.signal.App.loadManager(App.java:355)
	at org.asamk.signal.App.handleLocalCommand(App.java:288)
	at org.asamk.signal.App.init(App.java:229)
	at org.asamk.signal.Main.main(Main.java:60)
Caused by: java.lang.Exception: No native library found for os.name=Linux, os.arch=x86_64, paths=[/org/sqlite/native/Linux/x86_64:/usr/lib64:/lib64:/lib:/usr/lib]
	at org.sqlite.SQLiteJDBCLoader.loadSQLiteNativeLibrary(SQLiteJDBCLoader.java:375)
	at org.sqlite.SQLiteJDBCLoader.initialize(SQLiteJDBCLoader.java:68)
	at org.sqlite.core.NativeDB.load(NativeDB.java:63)
	at org.sqlite.SQLiteConnection.open(SQLiteConnection.java:278)
	... 29 common frames omitted
```
I could find a similar issue for Raspberry Pi, but this is a 64 bit x86 VM.
Version installed is `signal-cli-native/signalcli,now 0.11.6-1 amd64`

--- a/lib/build.gradle.kts
+++ b/lib/build.gradle.kts
@@ -19,7 +19,7 @@ dependencies {
     implementation("com.google.protobuf", "protobuf-javalite", "3.21.12")
     implementation("org.bouncycastle", "bcprov-jdk15on", "1.70")
     implementation("org.slf4j", "slf4j-api", "2.0.6")
-    implementation("org.xerial", "sqlite-jdbc", "3.40.0.0")
+    implementation("org.xerial", "sqlite-jdbc", "3.40.1.0")
     implementation("com.zaxxer", "HikariCP", "5.0.1")
 
     testImplementation("org.junit.jupiter", "junit-jupiter", "5.9.2")

